.env.dev

```diff
@@ -28,8 +28,6 @@ CLICKHOUSE_URL=http://localhost:8123
 CLICKHOUSE_USER=
 CLICKHOUSE_PASSWORD=dev_password_only
 
-# The URL to the CHAP CLI API that is used by the /upload endpoint to make predictions
-CHAP_CLI_API_URL=http://localhost:9697
 
-# Chap Configuration used by the /predict endpoint
+CHAP_CLI_API_URL=http://localhost:9697
 CHAP_URL=http://localhost:8000
```

Comment:
We need to decide on which to use here.

Reply:
The URLs reference different CHAP services, 
The env CHAP_CLI_API (CHAP CLI Service) is used when hitting the /uploads endpoint and uses a set of 3 files to generate a prediction.

The env CHAP_URL (CHAP Core Service) is used when hitting the /predict endpoint